homeassistant/components/sensor/ring.py

```diff
@@ -17,6 +17,7 @@
     CONF_ENTITY_NAMESPACE, CONF_MONITORED_CONDITIONS,
     STATE_UNKNOWN, ATTR_ATTRIBUTION)
 from homeassistant.helpers.entity import Entity
+from homeassistant.util.icon import icon_for_battery_level
```

Comment:
This should not have been in util. Util are helper methods that are not Home Assistant aware while this helper obviously is.

Please move it to `homeassistant.helpers.icon`

Reply:
`icon_for_battery_level` is aware of homeassistant. That should be inside util...?

Reply:
I agree now. It should be a helper

Reply:
@balloob @pvizeli guys do you want a new PR for moving the `icon_for_battery_level` to helpers or can I include on this one and change all the references to it?

Reply:
oh.. it seems the @happyleavesaoc  will do this weekend. Once his PR is merged, I'll modify this too then. thanks guys

Reply:
@happyleavesaoc had want this on last weekend...

Reply:
https://github.com/home-assistant/home-assistant/pull/9708

Sorry about the delay.